nodejs/week3/meal-app/src/backend/api/meals.js

```diff
@@ -0,0 +1,169 @@
+const express = require("express");
+const app = express();
+const router = express.Router();
+const pool = require("./../database");
+
+
+router.get("/", (req, res) => {
+    const maxPrice = req.query.maxPrice;
+    const availableReservations = req.query.availableReservations;
+    const title = req.query.title;
+    const createdAfter = req.query.createdAfter;
+    const limit = req.query.limit;
+
+    if (Object.entries(req.query).length === 0) {
+        pool.query("select * from meal", function(error, results, fields) {
```

Comment:
The overall approach in this route handler is good but it doesn't support filtering by multiple query parameters, it only supports 1. How could you support any combination of the filters?

Reply:
Hi orhan, could you please help me in how to support filtering by multiple query parameters. I couldnt figure it out how to do that